kde_ebm/plotting/plotting.py

```diff
@@ -19,14 +19,15 @@ def greedy_ascent_trace(greedy_dict):
 
 
 def mixture_model_grid(X, y, mixtures,
-                       score_names=None, class_names=None):
+                       score_names=None, class_names=None, hist_align='left'):
     n_particp, n_biomarkers = X.shape
     if score_names is None:
         score_names = ['BM{}'.format(x+1) for x in range(n_biomarkers)]
     if class_names is None:
         class_names = ['CN', 'AD']
     n_x = np.round(np.sqrt(n_biomarkers)).astype(int)
     n_y = np.ceil(np.sqrt(n_biomarkers)).astype(int)
+    hist_c = colors[:2]
```

Comment:
If colors is a constant then you should use capitals. 

https://www.python.org/dev/peps/pep-0008/#:~:text=(see%20below).-,Constants,Examples%20include%20MAX_OVERFLOW%20and%20TOTAL.